{"uid":"4b4757e66a1912dae1a509f688f20b0f","name":"categories","children":[{"name":"Product defects","children":[{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 450, in set_control_state\n    self.traffic_item.transmit(request_payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/trafficitem.py\", line 1307, in transmit\n    self._api._traffic.Apply()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/testplatform/sessions/ixnetwork/traffic/traffic.py\", line 1395, in Apply\n    return self._execute(\"apply\", payload=payload, response_object=None)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/base.py\", line 452, in _execute\n    response = self._connection._execute(url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 289, in _execute\n    return self._send_recv(\"POST\", url, payload)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 572, in _send_recv\n    return self._poll()\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 640, in _poll\n    return self._process_response_status_code(\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/.env/lib/python3.10/site-packages/ixnetwork_restpy/connection.py\", line 476, in _process_response_status_code\n    raise BadRequestError(message, response.status_code)\n  Error in L2/L3 Traffic Apply","children":[{"name":"test_vxlan_b2b","uid":"988edc664c26208f","parentUid":"ef6804d4159127be406a6111d9b6a662","status":"failed","time":{"start":1742299164935,"stop":1742299319142,"duration":154207},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"ef6804d4159127be406a6111d9b6a662"},{"name":"assert 0 == 1\n +  where 0 = len(<ixnetwork_restpy.testplatform.sessions.ixnetwork.availablehardware.chassis.chassis.Chassis object at 0x7f8e6830bc40>)","children":[{"name":"test_counter_ip_dscp","uid":"2c4ca17bc53b6e05","parentUid":"218b32e0354d8f2431f5d490ae2b0fe9","status":"failed","time":{"start":1742299164935,"stop":1742299187031,"duration":22096},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"218b32e0354d8f2431f5d490ae2b0fe9"},{"name":"snappi_ixnetwork.exceptions.SnappiIxnException:   File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/snappi_api.py\", line 584, in get_metrics\n    response = self.protocol_metrics.results(request)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 445, in results\n    return self._filter_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 418, in _filter_stats\n    return self._get_per_device_group_stats(protocol)\n  File \"/home/github-runner/actions-runner/_work/snappi-ixnetwork/snappi-ixnetwork/snappi_ixnetwork/protocolmetrics.py\", line 314, in _get_per_device_group_stats\n    raise Exception(msg)\n \n                Could not retrive drill down view                 at row index 0 Failed to get the View.href property as the View object has no encapsulated resources. Either the View.find() method was not called or it failed to retrieve any resources from the server.","children":[{"name":"test_encrypt","uid":"23b7f54cb267baab","parentUid":"c30c85a6c46c7e04bf2115be7873f128","status":"failed","time":{"start":1742299164935,"stop":1742299225608,"duration":60673},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"c30c85a6c46c7e04bf2115be7873f128"},{"name":"assert 2 == 1\n +  where 2 = len(<snappi.snappi.MkaMetricIter object at 0x7f8e6817c740>)\n +    where <snappi.snappi.MkaMetricIter object at 0x7f8e6817c740> = <snappi.snappi.MetricsResponse object at 0x7f8e67242520>.mka_metrics","children":[{"name":"test_encrypt_with_mka","uid":"944e754fa4c251e0","parentUid":"26f822832e1d5581fdcb536b3a0aea1f","status":"failed","time":{"start":1742299164935,"stop":1742299254694,"duration":89759},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"26f822832e1d5581fdcb536b3a0aea1f"},{"name":"Exception: Time out occurred while waiting for stats to be as expected","children":[{"name":"test_mka_stats","uid":"a06e45ce9dcea33c","parentUid":"3596725c47ed555d65da3fbf3423e1cd","status":"failed","time":{"start":1742299164935,"stop":1742299311326,"duration":146391},"flaky":true,"newFailed":true,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":[]}],"uid":"3596725c47ed555d65da3fbf3423e1cd"}],"uid":"8fb3a91ba5aaf9de24cc8a92edc82b5d"}]}